integration-libs/cdc/core/store/effects/cdc-user-addresses.effect.spec.ts

```diff
@@ -215,34 +454,40 @@ describe('CDC User Addresses effect', () => {
     });
   });
 
-  xdescribe('updateDefaultAddressInCDC', () => {
-    it('should invoke CDC JS service', () => {
-      cdcJSService.updateAddressWithoutScreenSet = jasmine
-        .createSpy()
-        .and.returnValue(of({ status: 'OK' }));
-      cdcUserAddressesEffect.updateDefaultAddressInCDC();
-      expect(cdcJSService.updateAddressWithoutScreenSet).toHaveBeenCalledWith(
-        mockUserAddress.formattedAddress,
-        mockUserAddress.postalCode,
-        mockUserAddress.town,
-        mockCountry.name
+  describe('updateDefaultAddressInCDC', () => {
+    it('should invoke CDC JS service', (done) => {
+      cdcJSService.updateAddressWithoutScreenSet = createSpy().and.returnValue(
+        of({
+          status: 'OK',
+        })
       );
+      cdcUserAddressesEffect.updateDefaultAddressInCDC();
+      cdcUserAddressesEffect.updateDefaultAddressInCDC().subscribe(() => {
```

Comment:
Do we _really_ need to call `cdcUserAddressesEffect.updateDefaultAddressInCDC()` 2 times in a row? I guess we can remove the first invocation.

Note: can be fixed after 6.0 FF